goclean.sh

```diff
@@ -5,14 +5,19 @@
 # 3. golint        (https://github.com/golang/lint)
 # 4. go vet        (http://golang.org/cmd/vet)
 # 5. test coverage (http://blog.golang.org/cover)
+#
+# gometalint (github.com/alecthomas/gometalinter) is used to run each each
+# static checker.
 
 set -e
 
 # Automatic checks
-test -z "$(gofmt -l -w .     | tee /dev/stderr)"
-test -z "$(goimports -l -w . | tee /dev/stderr)"
-test -z "$(golint .          | tee /dev/stderr)"
-go vet ./...
+test -z "$(gometalinter --disable-all \
+--enable=gofmt \
+--enable=golint \
+--enable=vet \
+--enable=goimports \
+--deadline=120s ./... | grep -v 'ALL_CAPS\|OP_\|ExampleNew' 2>&1 | tee /dev/stderr)"
```

Comment:
I removed the ALL_CAPS and OP_ though since those were specific to part of btcd.